agoradatatools/etl/transform.py

```diff
@@ -182,146 +164,105 @@ def transform_rna_seq_data(datasets: dict, adjusted_p_value_threshold: int):
-
-    gene_metadata = gene_metadata[
-        ['ensembl_gene_id', 'name', 'summary', 'symbol', 'alias', 'igap', 'eqtl', 'rna_in_ad_brain_change',
-         'rna_brain_change_studied', 'protein_in_ad_brain_change', 'protein_brain_change_studied']]
-    gene_metadata.rename(columns={'symbol': 'hgnc_symbol'}, inplace=True)
-
-    return gene_metadata
-
-
-def transform_gene_info(datasets: dict):
-
-    gene_metadata = datasets['gene_metadata']
-    target_list = datasets['target_list']
-    median_expression = datasets['median_expression']
-    druggability = datasets['druggability']
-
+    proteomics_concat.rename(columns = {'ensg': 'ensembl_gene_id'}, 
+                             inplace = True)
+    
     # these are the interesting columns of the druggability dataset
     useful_columns = ['geneid', 'sm_druggability_bucket', 'safety_bucket', 'abability_bucket', 'pharos_class',
                       'classification', 'safety_bucket_definition', 'abability_bucket_definition']
     druggability = druggability[useful_columns]
 
     target_list = nest_fields(df=target_list,
                               grouping='ensembl_gene_id',
-                              new_column='nominated_target')
+                              new_column='nominatedtarget')
 
     median_expression = nest_fields(df=median_expression,
                                     grouping='ensembl_gene_id',
-                                    new_column='median_expression')
+                                    new_column='medianexpression')
 
     druggability = nest_fields(df=druggability,
                                grouping='geneid',
                                new_column='druggability')
     druggability.rename(columns={'geneid': 'ensembl_gene_id'}, inplace=True)
 
-    for dataset in [target_list, median_expression, druggability]:
+    # Merge all the datasets
+    
+    for dataset in [igap, eqtl, rna_change, proteomics_concat, target_list, median_expression, druggability]:
         gene_metadata = pd.merge(left=gene_metadata,
                                  right=dataset,
                                  on='ensembl_gene_id',
-                                 how='left')
+                                 how='outer',
+                                 validate = 'one_to_one')
+    
+    # Populate values for rows that didn't exist in the individual datasets
+    
+    gene_metadata.fillna({'isIGAP': False,
+                          'haseqtl': False,
+                          'adj_p_val': -1,
+                          'cor_pval': -1}, inplace = True)
+    
+    # fillna doesn't work for creating an empty array, need this function instead
+    gene_metadata = fix_alias_field(gene_metadata)
+
+    gene_metadata['rna_brain_change_studied'] = (gene_metadata['adj_p_val'] != -1)
+    gene_metadata['isAnyRNAChangedInADBrain'] = (gene_metadata['adj_p_val'] <= adjusted_p_value_threshold)
```

Comment:
Ideally we should follow the standardized naming paradigm established by the framework, and use the agora_rename config option to adapt the field names to what agora expects. Over time, I'd like to align agora with the standard naming convention and not have to update the transform code when we do that.

Reply:
Undid this change so naming is snake_case as before. `agora_rename` field updated. (commit [1258ed40f91d6842e381cbaaac1eeef0d729c6cf](https://github.com/jaclynbeck-sage/agora-data-tools/commit/1258ed40f91d6842e381cbaaac1eeef0d729c6cf))